src/as/math/Vector3.spec.ts

```diff
@@ -530,7 +530,7 @@ describe('Vector3', () => {
 	// 	assert.equal(x.angleTo(z), Math.PI / 2)
 	// 	assert.equal(z.angleTo(x), Math.PI / 2)
 
-	// 	assert(Math.abs(x.angleTo(new Vector3(1, 1, 0)) - Math.PI / 4) < 0.0000001)
+	// 	expect(Math.abs(x.angleTo(new Vector3(1, 1).toBeTruthy()
```

Comment:
It looks like the regex might've gotten confused with this one. It's commented out right now.

Reply:
Awesome. Let me go fix this right now.